integration/docker_compose_integration_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+readonly DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+
+# Should be run from the root Trillian directory.
+docker_compose_up() {
+  local http_addr="$1"
+
+  docker-compose -f examples/deployment/docker-compose.yml up --build -d
+
+  # Wait until /healthz returns HTTP 200 and the text "ok", or fail after 60
+  # seconds. That should be long enough for the server to start.
+  health=$(wget --retry-connrefused --timeout 60 --output-document - \
+    "http://${http_addr}/healthz")
+  health_exitcode=$?
+  if [[ ${health_exitcode} = 0 ]]; then
+    echo "Health: ${health}"
+  fi
+
+  if [[ ${health_exitcode} != 0 && "${health}" != "ok" ]]; then
```

Comment:
Should this be `||`, or is this guaranteed to be exactly ok+0?

Reply:
It should, thanks - fixed.